src/design_docs/org/lib/actions.coffee

```diff
@@ -2,36 +2,49 @@ validate = require('./validation/validate')
 do_action = require('./shared/do_action')
 validate_doc_update = require('./shared/validate_doc_update').validate_doc_update
 h = require('./helpers')
+_ = require('./underscore')
 
 a = {}
 
 a.do_actions =
   team:
     'u+': (team, action, actor) ->
-      members = h.mk_objs(team.roles, [action.k, 'members'], [])
-      h.insert_in_place(members, action.v)
+      members = h.mk_objs(team.roles, [action.role, 'members'], [])
+      h.insert_in_place(members, action.user)
     'u-': (team, action, actor) ->
-      members = h.mk_objs(team.roles, [action.k, 'members'], [])
-      h.remove_in_place(members, action.v)
+      members = h.mk_objs(team.roles, [action.role, 'members'], [])
+      h.remove_in_place(members, action.user)
     'a+': (team, action, actor) ->
-      assets = h.mk_objs(team.rsrcs, [action.k, 'assets'], [])
-      h.insert_in_place(assets, action.v)
+      action.asset.id = action.id
```

Comment:
doAction (on the node.js side) will provide a default action id if you don't specify one.